Services/Auth0toAI/Auth0toAI/Auth0toAI.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+	  <PackageReference Include="Azure.Messaging.EventGrid" Version="4.15.0" />
+	  <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+	  <PackageReference Include="Microsoft.ApplicationInsights.Web" Version="2.21.0" />
+	  <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
+	  <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
+	  <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.13.0" />
+	  <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.EventGrid" Version="3.2.1" />
```

Comment:
Why Webjobs ?

[Isolated Worker V4](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#core-packages) guide doesn't mention it.